NoMansLand/Assets/Scripts/RandomSpawner.cs

```diff
@@ -48,10 +52,26 @@ IEnumerator CollectibleDrop()
         getRelevantSpaceshipParts(spaceship);
         for (int i = 0; i < spaceShipPartsToSpawn.Count; i++)
         {
-            // generate random X position in the range (terrainXPos + 50, terrainXPos + terrainXLength - 50)
-            itemXPos = UnityEngine.Random.Range(terrainXPos + cushionAmount, terrainXPos + terrainXLength - cushionAmount); 
-            // generate random Z position in the range (terrainZPos + 50, terrainZPos + terrainZLength - 50)
-            itemZPos = UnityEngine.Random.Range(terrainZPos + cushionAmount, terrainZPos + terrainZLength - cushionAmount);
+            // Basically: the "spawnable" region has been split into two areas  
+            // Area 1: edge of map (+ cushion amount) up until spaceship (- cushion amount)
+            // Area 2: spaceship ( + cushion amount) up until other edge of map 
+            // As a result, spaceship +/- cushion amount has been excluded from spawning items 
+            // We will spawn in these two areas equally 
+            if(i%2 == 0) 
+            {
+                // Generate an X position in Area 1 
+                itemXPos = UnityEngine.Random.Range(terrainXPos + cushionAmount, spaceshipX - cushionAmount); 
```

Comment:
Wondering if this algorithm will generate dead spots? Places where parts will not spawn again. reaching out to u on fb

Reply:
Update: Resolved on call